src/test/java/uk/gov/register/functional/db/RecordQueryFunctionalTest.java

```diff
@@ -91,6 +94,7 @@ public void setup() throws IOException {
     @After
     public void tearDown() {
         dbi.close(handle);
+        register.wipe();
```

Comment:
This test was previously not cleaning database state which caused subsequent tests to fail depending in what order tests were run.